test/NuGet.Core.Tests/NuGet.Common.Test/PathResolverTests.cs

```diff
@@ -650,54 +649,34 @@ private static IEnumerable<string> GetPlatformSpecificPaths(IEnumerable<string>
         file1.txt
         file2.txt
     */
-    public sealed class TestDirectoryFixture : IDisposable
+    public sealed class TestDirectoryFixture : TestDirectory
     {
-        private DirectoryInfo _directory;
-
-        public string TestDirectoryPath
-        {
-            get { return _directory.FullName; }
-        }
-
         public TestDirectoryFixture()
+            : base(System.IO.Path.Combine(System.IO.Path.GetTempPath(), System.IO.Path.GetRandomFileName()))
```

Comment:
Wait, `TestFileSystemUtility.CreateRandomTestFolder` returns a `TestDirectory` instance, which I do not want.  I want to subclass `TestDirectory`.